### DISCUSSION TITLE
Passing verification variables through Middleware

### DISCUSSION THREAD

**Discussion by thoughtfulviewfinder** [leafsphp/leaf#241] [Q&A]
Hello,
I am new to Leaf PHP but I am learning how to setup Middleware to run a JWT verification.

I have got the middleware running perfectly and exiting when the JWT verification fails. Now I would like to pass on the decoded JWT variables to the route, I am unable to understand how to pass it. I tried going through the Middleware documentation and didn't see a section describing this.  Could someone point me to the right direction?

Thanks,
Cheese.

**Comment by ibnsultan** [comment]
@mychidarko
